src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueBatchContainer.cs

```diff
@@ -102,6 +112,14 @@ public bool ImportRequestContext()
             return false;
         }
 
+        public void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
No, I think that would be wrong. Its a different sequence if from what is stored.
If the sequence id should not be stored, it should be marked `[NonSerialized]`.  
However, now it is serialized, so I would just preserve the same behavior.